singularity-opac/photons/gray_opacity_photons.hpp

```diff
@@ -43,37 +43,37 @@ class GrayOpacity {
   inline void Finalize() noexcept {}
 
   PORTABLE_INLINE_FUNCTION
-  Real AbsorptionCoefficientPerNu(const Real rho, const Real temp,
-                                  const Real nu, Real *lambda = nullptr) const {
+  Real AbsorptionCoefficient(const Real rho, const Real temp, const Real nu,
+                             Real *lambda = nullptr) const {
```

Comment:
Why this change? Do we no longer include the bolumetric/gray absorption coefficient in each opacity class?

Reply:
I took off the `PerNu` because unlike e.g. `EmissivityPerNu` there's no `d \nu` in the denominator here. It's still a frequency-dependent call to the absorption coefficient though. Should I go back?